spec/factories/system/api_accesses.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
```

Comment:
this file not created from me but just moved from the `fixtures` folder to 'fixtures/system'

and then renamed the fixture file name from 'api_access.rb' to its pluralize variant 'api_accesses.rb'

and this chnages made to allow annotate gem to detect fixture automatically